addons/web/static/src/views/view_components/report_view_measures.xml

```diff
@@ -3,14 +3,15 @@
 
     <t t-name="web.ReportViewMeasures">
       <Dropdown>
-        <button class="btn btn-primary">
-          Measures <i class="fa fa-caret-down ms-1"/>
+        <button class="btn btn-primary o_report_measures">
+          <t t-if="props.showActiveMeasure and props.activeMeasures.length" t-esc="props.measures[props.activeMeasures[0]].string"/>
```

Comment:
That's a bit weird (especially the combo `showActiveMeasure = true` + `activeMeasures.length > 1`.

Here's an alternative: instead of a `showActiveMeasure` props, add a `togglerLabel` props, and leave the responsibility to the pivot view to specify the label, which would be `model.metaData.measure`